README.md

```diff
@@ -249,14 +285,15 @@ Version 22 of Kubernetes collector allows you to collect cluster events and push
 
   kubectl describe configmaps -n kube-system plugin-configs task-manifests
   ```
-* Edit `configmaps/appoptics-token.yaml` and insert your `APPOPTICS_TOKEN`.
-* Create ServiceAccount and Deployment:
+* Create Kubernetes secret for APPOPTICS_TOKEN
```

Comment:
```suggestion
* Create Kubernetes secret for SOLARWINDS_TOKEN
```

?

Reply:
hm, to be discussed with @tomekkolo and @melord as there are many more occurrences

Reply:
yes, SOLARWINDS_TOKEN should be used everywhere

Reply:
generally, SOLARWINDS_TOKEN is the variable to be used everywhere, but for backward compatibility APPOPTICS_TOKEN should also be accepted inside container. Initi script should require either variable to be present https://github.com/solarwinds/solarwinds-snap-agent-docker/blob/1ec91aceb000de94f3b63eeb32be9c4ea8d58321/conf/swisnap-init.sh#L10

Reply:
Added handling SOLARWINDS_TOKEN and moved APPOPTICS_TOKEN to be used as an fallback option.